src/plots/gl2d/scene2d.js

```diff
@@ -169,6 +169,26 @@ proto.toImage = function(format) {
     this.stopped = true;
     if(this.staticPlot) this.container.appendChild(STATIC_CANVAS);
 
+
+
+    var glplot = this.glplot,
+        pixelRatio = this.pixelRatio,
+        fullLayout = this.fullLayout;
+
+    var width = fullLayout.width,
+        height = fullLayout.height,
+        pixelWidth = Math.ceil(pixelRatio * width) |0,
+        pixelHeight = Math.ceil(pixelRatio * height) |0;
+
+    // check for resize
+    var canvas = this.canvas;
+    if(canvas.width !== pixelWidth || canvas.height !== pixelHeight) {
```

Comment:
This is duplicate of https://github.com/dfcreative/plotly.js/blob/07af2b83e8d6b4bb9da5363ac27ffeeebadf0fd0/src/plots/gl2d/scene2d.js#L131 and https://github.com/dfcreative/plotly.js/blob/07af2b83e8d6b4bb9da5363ac27ffeeebadf0fd0/src/plots/gl2d/scene2d.js#L374. Should I put it into a method `updateSize`?